### ISSUE TITLE
Implement Age 8 cards for Artifacts Expansion

### ISSUE THREAD

**Issue by micahstairs** [micahstairs/bga-innovation#145]
This issue will track the implementation of all Age 8 cards for the Artifacts expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

  - [ ] **185 - Parnell Pitch Drop**
    - Status: NOT STARTED
  - [ ] **186 - Earhart's Lockheed Electra 10E**
    - Status: NOT STARTED
  - [ ] **187 - Battleship Bismarck**
    - Status: NOT STARTED
  - [ ] **188 - Battleship Yamato**
    - Status: NOT STARTED
  - [ ] **189 - Ocean Liner Titanic**
    - Status: NOT STARTED
  - [ ] **190 - Meiji-Mura Stamp Vending Machine**
    - Status: NOT STARTED
  - [ ] **191 - Plush Beweglich Rod Bear**
    - Status: NOT STARTED
  - [ ] **192 - Time**
    - Status: NOT STARTED
  - [ ] **193 - Garland's Ruby Slippers**
    - Status: NOT STARTED
  - [ ] **194 - '30 World Cup Final Ball**
    - Status: NOT STARTED

**Comment by ultimatefiend** [comment]
This is what I had for World Cup Ball before I realized that the transfer wouldn't work. :)

(194, 1, 8, 4, 2, 2, 6, 0, 2, FALSE, 0, 'deck', 0, NULL, FALSE),

            // id 194, Artifacts age 8: 30 World Cup Final Ball
            case "194C1":
                $step_max = 1;
                break;

            case "194N1":
                $happened = true;
                do {
                    // Draw and reveal an 8.
                    $card = self::executeDraw($player_id, 8, 'revealed');
                
                    // "The single player with the highest top card of the drawn card's color achieves it, ignoring eligibility. 
                    self::transferCardFromTo($card, $player_id, 'achievements');
                    
                    // If that happens, repeat this effect."
                    $happened = false;
                } while($happened);
                self::transferCardFromTo($card, $player_id, 'hand'); // unspoken : card is placed in hand
                break;




        // id 194, Artifacts age 8: 30 World Cup Final Ball
        case "194C1A":
            // "return one of your achievements!"
            $options = array(
                'player_id' => $player_id,
                'n' => 1,
                'can_pass' => false,

                'owner_from' => $player_id,
                'location_from' => 'achievements',
                'owner_to' => 0,
                'location_to' => 'deck'                
            );
            break;
